open_rarity/resolver/opensea_api_helpers.py

```diff
@@ -177,6 +241,71 @@ def get_collection_with_metadata(
     return collection_with_metadata
 
 
+def get_collection_from_opensea(
+    opensea_collection_slug: str,
+) -> Collection:
+    """Fetches collection and token data with OpenSea endpoint and API key
+    and stores it in the Collection object
+
+    Parameters
+    ----------
+    opensea_collection_slug : str
+        collection slug on opensea's system
+
+    Returns
+    -------
+    Collection
+        collection abstraction
+
+    """
+    # Fetch collection metadata
+    collection_obj = fetch_opensea_collection_data(
+        slug=opensea_collection_slug
+    )
+    contracts = collection_obj["primary_asset_contracts"]
+    interfaces = set([contract["schema_name"] for contract in contracts])
+    stats = collection_obj["stats"]
+    if not interfaces.issubset(set(["ERC721", "ERC1155"])):
+        raise ValueError(
+            "We currently do not support non EVM standards at the moment"
+        )
```

Comment:
Depending on how many times we use this exact error maybe we can make an `ERCStandardError` that derives from `ValueError` and has a default message.